terraform/modules/alertmanager/alertmanager-service.tf

```diff
@@ -161,6 +161,10 @@ data "pass_password" "notify_zendesk" {
   path = "receivers/notify/zendesk"
 }
 
+data "pass_password" "notify_p2_pagerduty_key" {
```

Comment:
On the naming of the key, is the key specific to P2 incidents, or is it not specific to the priority?

Reply:
It is specific to p2 incidents. We have one key for p1 incidents and one key for p2 incidents (although this PR only adds the p2 key as we don't yet require the p1 key in any of these alerts)

Reply:
Cool, good to know :+1: